tests/foreman/virtwho/test_api_virtwho.py

```diff
@@ -153,21 +175,19 @@ def test_positive_deploy_configure_by_script(self):
             (guest_name, 'product_id={} and type=STACK_DERIVED'.format(
                 self.vdc_physical))]
         for hostname, sku in hosts:
+            vdc_id = None
```

Comment:
Same as above: `vdc_id = None` is not necessary.

Reply:
yes, thanks